src/pip/_internal/utils/glibc.py

```diff
@@ -4,8 +4,14 @@
 import re
 import warnings
 
+from pip._internal.utils.typing import MYPY_CHECK_RUNNING
+
+if MYPY_CHECK_RUNNING:
+    from typing import Tuple  # noqa: F401
+
 
 def glibc_version_string():
+    # type: () -> str
```

Comment:
This should be `type: () -> Optional[str]`

Reply:
Ha, missed it - you are right. In process of rebasing, will include.